Update datatable-customization.mdx (#3757)
grommet/hpe-design-system: aries-site/src/pages/templates/datatable-customization.mdx

and a CheckBoxGroup that lists all possible table columns.

When a column in the list is checked, it is visible in the data table. When it is unchecked,

> it is removed from the data table. When a column from the data table is perceived as mandatory and should not be unchecked, display it as checked and disabled. Adjustments to the table data should occur immediately as

Reviewer: ```suggestion
it is removed from the data table. When a column from the data table is required, display it as checked and disabled. Adjustments to the table data should occur immediately as
```
